### PR TITLE
Case 21993: Globally disallow use of the camera or microphone by hosted web content

### PR DESCRIPTION
**Pull request by jherico**
[21993](https://highfidelity.manuscript.com/f/cases/21993/Security-Issue-disable-all-access-to-webcams-and-microphones-for-web-surfaces-of-any-kind)

### CODE REVIEW

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -2877,11 +2877,19 @@ void Application::initializeGL() {
...
 #if !defined(DISABLE_QML)
+    QStringList chromiumFlags;
+    // Bug 21993: disable microphone and camera input 
+    chromiumFlags << "--use-fake-device-for-media-stream";
     // Disable signed distance field font rendering on ATI/AMD GPUs, due to
     // https://highfidelity.manuscript.com/f/cases/13677/Text-showing-up-white-on-Marketplace-app
     std::string vendor{ (const char*)glGetString(GL_VENDOR) };
     if ((vendor.find("AMD") != std::string::npos) || (vendor.find("ATI") != std::string::npos)) {
-        qputenv("QTWEBENGINE_CHROMIUM_FLAGS", QByteArray("--disable-distance-field-text"));
+        chromiumFlags << "--disable-distance-field-text"
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: missing `;`

**Comment by conklin94122** [comment]
Please test in the v0.81.1 release PR

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18507/HighFidelity-Beta-PR15299-50d2844.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18507/HighFidelity-Beta-Interface-PR15299-50d2844.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18507/HighFidelity-Beta-PR15299-50d2844.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18507/HighFidelity-Quest-Beta-PR15299-50d2844.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18507/HighFidelity-Beta-PR15299-50d2844.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18507/HighFidelity-Beta-Interface-PR15299-50d2844.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18510/HighFidelity-Beta-PR15299-03bf5b5.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18510/HighFidelity-Beta-Interface-PR15299-03bf5b5.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18510/HighFidelity-Beta-PR15299-03bf5b5.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18510/HighFidelity-Quest-Beta-PR15299-03bf5b5.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18510/HighFidelity-Beta-PR15299-03bf5b5.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18510/HighFidelity-Beta-Interface-PR15299-03bf5b5.exe

**Comment by ZappoMan** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18511/HighFidelity-Beta-PR15299-03bf5b5.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18511/HighFidelity-Quest-Beta-PR15299-03bf5b5.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18511/HighFidelity-Beta-PR15299-03bf5b5.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18511/HighFidelity-Beta-Interface-PR15299-03bf5b5.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18511/HighFidelity-Beta-PR15299-03bf5b5.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18511/HighFidelity-Beta-Interface-PR15299-03bf5b5.exe

**Comment by conklin94122** [comment]
Still in testing but need to get this into the release branch ASAP.

**Pull request merged**